### ISSUE TITLE
Numpy reshape function returns values in a misplaced order

### ISSUE THREAD

**Issue by ahoelzemann** [numpy/numpy#16680]
Hi everyone, 

I'm not sure if I found a bug in the reshape function, or if I have a mistake in my code. Please have a look on the attached two snippets below. I'm scaling 3D IMU-Data axis wise between -1 and 1, To do so I iterate over the three axis (signal) and scale them separately. After scaling them, I cast my list to a numpy array. This array has the dimension (x, y, 1), but I need my array in the shape y, x. That's why I transposed it and then try to delete the unnecessary third dimension. In the first example I did this by using the reshape function in the second example I just took the first element from the first dimension. Surprisingly I get different results. It seems like reshape is misplacing the samples. Around index 6000 you can see some clear changes.

Please have a closer look on the two plots below.
![correct_order](https://user-images.githubusercontent.com/24317680/85626452-5dd2ad80-b66d-11ea-881b-7947058076c9.png)
![misplaced_order](https://user-images.githubusercontent.com/24317680/85626454-5e6b4400-b66d-11ea-859b-fcdb470ae35f.png)

### Reproducing code example:

```python
import numpy as np
result = []
        for signal in signals:
            temp_signal = scale_between_minus_one_and_one(signal)
            result.append(temp_signal)
        result = np.array(result)
        result = result.T
        result = np.reshape(result, newshape=(result.shape[1], result.shape[2]))
```
And 

```python
import numpy as np
result = []
        for signal in signals:
            temp_signal = scale_between_minus_one_and_one(signal)
            result.append(temp_signal)
        result = np.array(result)
        result = result.T[0,:,:]
```
```python
def scale_between_minus_one_and_one(input):
    from sklearn.preprocessing import MinMaxScaler
    
    return MinMaxScaler(feature_range=(-1,1)).fit_transform(input)
```

### Numpy/Python version information:
Python 3.8 and Numpy 1.18.4

**Comment by ahoelzemann** [comment]
Since in the second plot the z-axis is slightly lower than in the first, I think that the mistake is somewhere in my pre-precossing steps.
